app/controllers/cases_controller.rb

```diff
@@ -0,0 +1,89 @@
+class CasesController < ApplicationController
+  before_action :set_case, only: [:show,:edit,:update,:destroy]
+  def index
+    @cases=Case.all
+  end
+
+  def show
+  end
+
+  def update
+    respond_to do |format|
+      if @case.update(case_params)
```

Comment:
Can you please explain me why you have used `respond_to` even if there is only one format? Please refer: [reference-1](http://api.rubyonrails.org/classes/ActionController/MimeResponds.html) [reference-2](http://stackoverflow.com/questions/9492362/rails-how-does-the-respond-to-block-wor)